cloudslang-tests/src/test/java/io/cloudslang/lang/systemtests/ParallelLoopFlowsTest.java

```diff
@@ -49,6 +55,30 @@ public void testFlowWithParallelLoop() throws Exception {
     }
 
     @Test
+    public void testFlowWithSensitiveInputParallelLoop() throws Exception {
+        URI resource = getClass().getResource("/yaml/loops/parallel_loop/sensitive_input_parallel_loop.sl").toURI();
+        URI operation1 = getClass().getResource("/yaml/loops/parallel_loop/print_branch.sl").toURI();
+        Set<SlangSource> path = Sets.newHashSet(SlangSource.fromFile(operation1));
+
+        RuntimeInformation runtimeInformation = triggerWithData(SlangSource.fromFile(resource), path);
+
+        List<StepData> branchesData = extractParallelLoopData(runtimeInformation);
+        Assert.assertEquals("incorrect number of branches", 3, branchesData.size());
```

Comment:
isn't these tests suppose to validate the some data  - like the flow inputs/outputs are sensitive?